### PR TITLE
Documentation page

### PR DESCRIPTION
**Pull request by Thomasalex2**
- Made modifications and tweaks to components in the library (work still needs to be done for further generalizations)
- Added scroll animations to all links and buttons
- Colour scheme is NOT FINAL. So you can mostly ignore that
- Added alerts, badges, buttons, forms, cards, images to the documentation page

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -8,6 +8,7 @@
     --secondary-color: #42919E;
     --black-text-color: black;
```
Comment (deekshithmd) [inline styles.css]: using color codes will be good

File: resources/fervent-ui/Components/forms.css [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+select:focus {
+    border: 2px solid var(--primary-color);
+}
+
+textarea {
+    display:block;
+    min-width: 50%;
+    max-width: 100%;
+    min-height: 50px;
```
Comment (deekshithmd) [inline resources/fervent-ui/Components/forms.css]: check about using rem here...

File: resources/fervent-ui/Components/badge.css [hunk 3]
```diff
@@ -25,4 +37,30 @@
...
+}
+
+.avatar-bdg {
+    height: 15%;
+    width: 15%;
+    border-radius: 50%;
+    border: 4px solid white;
+    display: inline-block;
+    position: absolute;
+    bottom: 5px;
```
Comment (AshutoshM29) [inline resources/fervent-ui/Components/badge.css:49]: Just a small suggestion, try using rem instead of px
rest looks good!!

**Review by AshutoshM29 — CHANGES_REQUESTED** [review]

File: resources/fervent-ui/Components/badge.css (hunk 3 above)
Comment (Thomasalex2) [reply resources/fervent-ui/Components/badge.css:49]: I think I read somewhere that if you are using units less than 10px, its okay to not use rems. But thank you for the suggestion anyway. I'll look it up even more

File: resources/fervent-ui/Components/forms.css (hunk 2 above)
Comment (Thomasalex2) [reply resources/fervent-ui/Components/forms.css]: Ahh thanks. Missed that

**Comment by Thomasalex2** [comment]
- Suggestions have been incorporated
- The website is feature complete
- Covers all of P0 and some of P1 and P2

**Pull request merged**